src/browser/monkey-menu.css

```diff
@@ -177,34 +176,38 @@ section {
   text-decoration: none;
 }
 
+.subview-item .icon::before {
+  vertical-align: middle;
+}
+
 .subview-item .icon {
   font-size: 14px;
+  flex-shrink: 0;
   text-align: center;
-  width: 19px;
+  max-height: 16px;
+  width: 16px;
 }
 
 .subview-item .icon img {
-  object-fit: contain;
```

Comment:
This is what makes the icons retain the correct aspect ratio.  Are they still, when this is removed?

Reply:
![](https://eselce.github.io/GitTest/misc/img/GM-menu-style-box.png)

Reply:
> This is what makes the icons retain the correct aspect ratio. Are they still, when this is removed?

Yes. As shown by @Eselce . And here's another image from a system that's not inflicted with the offset disease.

![scaling](https://user-images.githubusercontent.com/6609453/36638494-6aa661fa-19bc-11e8-8c6d-d16a8317c92f.png)

Reply:
Scratch that. No it doesn't. Going to readd it shortly. Need testing from @Eselce to see if it screws up the lines.

Reply:
Damned: 1:2

Reply:
Lol, 2:2...

Reply:
Args, 3:3...

![](https://eselce.github.io/GitTest/misc/img/GM-menu-style-box2.png)

Reply:
3:3 Overtime for Gold... Tell me, how to create that menu shot with such a distraction! :smile:

Reply:
It would be a good idea to add two test scripts: one with an icon that is simply very large (wide and tall), one that is very wide (but not very tall).  And check both those cases.

Reply:
Wide is a long script name, but what is tall? Capital letters?

Reply:
I think he meant the aspect ratio of the icon itself.

Reply:
My bad. Didn't read it carefully enough: "one with an **icon**..."
Does anyone have a script with such two icons?

Reply:
This is a little arbitrary, but...
![](https://eselce.github.io/GitTest/misc/img/GM-menu-style-box3.png)

Reply:
The latter icon is essentially of square shape (aspect ratio 1.0), the former is wider than the height.
Don't know, if it make sense to use icons of big data size (should be the same with huge icons of same aspect ratio)...

Reply:
@Eselce , assuming those icons are free for use, can you make a PR adding them to the test scripts folder? If data URIs work for the `@icon` (if they don't then that should be addressed) then they can probably be included as such rather than separate files.

Reply:
Hmm., these were just arbitrary images, that I could find, maybe others are more useful.
They can be find on GitHub:

https://eselce.github.io/OS2.scripts/img/tor.png
https://eselce.github.io/OS2.scripts/img/sch.png

Hmm, funny, I thought, the goal image was not of square shape...

Reply:
There are probably better images available. Maybe I should do that test again!